src/pkg/securityhub/dao/security.go

```diff
@@ -0,0 +1,133 @@
+// Copyright Project Harbor Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dao
+
+import (
+	"context"
+
+	"github.com/goharbor/harbor/src/lib/orm"
+	"github.com/goharbor/harbor/src/lib/q"
+	"github.com/goharbor/harbor/src/pkg/scan/dao/scan"
+	"github.com/goharbor/harbor/src/pkg/securityhub/model"
+)
+
+const (
+	summarySQL = `select sum(s.critical_cnt) critical_cnt,
+       sum(s.high_cnt)     high_cnt,
+       sum(s.medium_cnt)   medium_cnt,
+       sum(s.low_cnt)      low_cnt,
+       sum(s.none_cnt)     none_cnt,
+       sum(s.unknown_cnt)  unknown_cnt,
+       sum(s.fixable_cnt)  fixable_cnt
+from artifact a
+         left join scan_report s on a.digest = s.digest
+         where s.registration_uuid = ?`
+
+	dangerousArtifactSQL = `select a.project_id project, a.repository_name repository, a.digest, s.critical_cnt, s.high_cnt, s.medium_cnt, s.low_cnt
+from artifact a,
+     scan_report s
+where a.digest = s.digest
+  and s.registration_uuid = ?
+order by s.critical_cnt desc, s.high_cnt desc, s.medium_cnt desc, s.low_cnt desc
+limit 5`
+
+	scannedArtifactCountSQL = `select count(1) 
+           from artifact a 
+      left join scan_report s on a.digest = s.digest 
+          where s.registration_uuid= ? and s.uuid is not null`
+
+	dangerousCVESQL = `select vr.*
+from vulnerability_record vr
+where vr.cvss_score_v3 is not null
+and vr.registration_uuid = ?
+order by vr.cvss_score_v3 desc
+limit 5`
```

Comment:
We can add some comments here to explain background and concerns why limit it to 5.